src/OpenLoco/Objects/ObjectManager.cpp

```diff
@@ -74,9 +74,7 @@ namespace OpenLoco::ObjectManager
     assert_struct_size(ObjectRepositoryItem, 8);
 #pragma pack(pop)
 
-    loco_global<ObjectEntry2[maxObjects], 0x1125A90> objectEntries;
     loco_global<ObjectRepositoryItem[64], 0x4FE0B8> object_repository;
-    loco_global<Object* [maxObjects], 0x0050C3D0> _allObjects;
```

Comment:
Are these two removed globals now completely obsolete?

Reply:
Yes I'm trying to avoid accessing through this interface.

Reply:
I simplified it a bit more to make it clearer.